src/codex_plus/request_logger.py

```
@@ -0,0 +1,91 @@
+
+            async with aiofiles.open(log_file, 'w') as f:
+                await f.write(payload_content)
+            logger.info(f"Logged full payload to {log_file}")
+
+            # Also log instructions if available
+            if "instructions" in payload and isinstance(payload["instructions"], str):
+                instructions_file = log_dir / "instructions.txt"
+                async with aiofiles.open(instructions_file, 'w') as f:
+                    await f.write(payload["instructions"])
+                logger.info(f"Logged instructions to {instructions_file}")
+        except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)